src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -9,11 +9,7 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
 import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
 
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
By the way we cannot import all libraries using * due to the guidelines mentioned in the course.